CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 18.0.0 Adding job search version 3 as additional call
```

Comment:
The arguments should be backwards compatible.  The results may not be exactly.  So we wanted to be able to switch between endpoints.  Eventually we should be moving to this endpoint exclusively.  None of that however necessarily breaks the consumer of this API.  So, I guess we should only up the minor version?